Xamarin.Forms.Platform.GTK/Renderers/ButtonRenderer.cs

```diff
@@ -45,6 +46,7 @@ protected override void OnElementChanged(ElementChangedEventArgs<Button> e)
                 UpdateText();
                 UpdateBorder();
                 UpdateContent();
+				UpdateTextAlignment();
```

Comment:
Whitespace

Reply:
@llevera It's been a while... I've tried this with images alongside text, and the images also get aligned :/ Not sure how to implement this otherwise.